src/frontend/frontend.ts

```diff
@@ -1,31 +1,23 @@
-import {Component, Inject, bind, NgZone} from '@angular/core';
-
+import {Component, NgZone} from '@angular/core';
 import {bootstrap} from '@angular/platform-browser-dynamic';
-
 import {Dispatcher} from './dispatcher/dispatcher';
-
 import {BackendActions} from './actions/backend-actions/backend-actions';
 import {UserActions} from './actions/user-actions/user-actions';
-
-import {UserActionType}
-  from './actions/action-constants';
-
-import {ComponentDataStore}
-  from './stores/component-data/component-data-store';
-
+import {UserActionType} from './actions/action-constants';
+import {ComponentDataStore} from './stores/component-data/component-data-store';
 import {BackendMessagingService} from './channel/backend-messaging-service';
-
 import {TreeView} from './components/tree-view/tree-view';
 import {InfoPanel} from './components/info-panel/info-panel';
 import AppTrees from './components/app-trees/app-trees';
 import {Header} from './components/header/header';
-
 import * as Rx from 'rxjs';
-
 import {ParseUtils} from './utils/parse-utils';
 
 const BASE_STYLES = require('!style!css!postcss!../styles/app.css');
 
+// (ericjim) tweak these values based off research on the stress tester.
+const ALLOWED_DEPTH: number = 3; // assume a non-complex tree and reveal all nodes in the component tree.
```

Comment:
@igor-ka as you suggested, I'm defaulting the depth of the component tree to 3. The reasoning was to retain consistency across all apps (large and small). I've attached a zip containing the crx file. 
[augury.crx.zip](https://github.com/rangle/augury/files/372169/augury.crx.zip)